eta/models/manifest.json

```diff
@@ -917,15 +917,47 @@
             "requirements": {
                 "cpu": {
                     "support": true,
-                    "packages": ["tensorflow<2"]
+                    "packages": ["tensorflow"]
                 },
                 "gpu": {
                     "support": true,
-                    "packages": ["tensorflow-gpu<2"]
+                    "packages": ["tensorflow-gpu"]
                 }
             },
             "date_added": "2019-04-11 12:25:44"
         },
+        {
+            "base_name": "centernet-hg105-512-coco-tf2",
+            "base_filename": "centernet_hg104_512x512_coco17_tpu-8.tar.gz",
+            "version": null,
+            "description": "CenterNet architecture from `Objects as Points <https://arxiv.org/abs/1904.07850>`_ with the hourglass backbone trained on COCO",
+            "manager": {
+                "type": "eta.core.models.ETAModelManager",
+                "config": {
+                    "url": "http://download.tensorflow.org/models/object_detection/tf2/20200713/centernet_hg104_512x512_coco17_tpu-8.tar.gz",
+                    "extract_archive": true,
+                    "delete_archive": true
+                }
+            },
+            "default_deployment_config_dict": {
+                "type": "eta.detectors.TF2ModelsDetector",
+                "config": {
+                    "model_name": "centernet-hg105-512-coco-tf2",
+                    "labels_path": "{{eta}}/tensorflow/models/research/object_detection/data/mscoco_complete_label_map.pbtxt"
+                }
+            },
+            "requirements": {
+                "cpu": {
+                    "support": true,
+                    "packages": ["tensorflow>=2"]
+                },
+                "gpu": {
+                    "support": true,
+                    "packages": ["tensorflow-gpu>=2"]
```

Comment:
Didn't they get rid of `tensorflow-gpu` starting in TF2? I think you may be able to do this instead:
```
            "requirements": {
                "packages": "tensorflow>=2",
                "cpu": {
                    "support": true,
                },
                "gpu": {
                    "support": true
                }
            }
```

Reply:
Well would you look at that, this is the first I've heard of it: https://www.tensorflow.org/install/gpu#pip_package